src/components/card/Card.js

```diff
@@ -17,17 +31,57 @@ function Card(props) {
           ) : (
             <img className="deceased" src={deceased} alt="deceased" />
           )}
```

Comment:
I can't comment on the code above this line but basically, I wouldn't do it that way. I will try and explain. So this is what you are doing:
```javascript
{dead ? (
            console.log(true)
          ) : (
            <img className="deceased" src={deceased} alt="deceased" />
          )}
```
I would write it like this:
```javascript
{!isAlive && <img className="deceased" src={deceased} alt="deceased" />}
```
With that I say if `isAlive` is not true, then render the image. Does that make sense?
The `console.log` in your solution is not necessary and not considered best practice. If you have any questions about that, please get back to me.